MovieKunoUITests/MovieKunoUITests.swift

```diff
@@ -8,35 +8,41 @@
 import XCTest
 
 class MovieKunoUITests: XCTestCase {
+    
+    let app = XCUIApplication()
```

Comment:
Screenshots should be included in the commit, since that's the purpose of its functionality to have an overview of the layout of the project, maybe create a folder for it.